arbor/cable_cell_group.cpp

```diff
@@ -215,9 +210,9 @@ void run_samples(
     fvm_probe_scratch& scratch)
 {
     const sample_size_type n_raw_per_sample = p.raw_handles.size();
-    sample_size_type n_sample = (sc.end_offset-sc.begin_offset)/n_raw_per_sample;
-    arb_assert((sc.end_offset-sc.begin_offset)==n_sample*n_raw_per_sample);
-    arb_assert((unsigned)n_raw_per_sample==p.weight.size());
+    sample_size_type n_sample = (sc.end_offset - sc.begin_offset)/n_raw_per_sample;
+    arb_assert((sc.end_offset - sc.begin_offset)==n_sample*n_raw_per_sample);
+    arb_assert((unsigned)n_raw_per_sample == p.weight.size());
```

Comment:
Are these changes to make CodeQL happy, or just for readability?

Reply:
Readability. I can get behind `a*b + c` for similarities with mathematical notation, but `=`, `==`, and especially `-` tend to merge visually into one token. Or: Too similar to `snake_case` for my liking.
As much of the code, especially newer parts is in this style, I tend to adjust these expression in passing when editing files. Same for removing redundant headers, which my IDE picks up on these days.